modules/fluid_properties/fluid_properties.mk

```
@@ -0,0 +1,7 @@
+# Saline
+SALINE_DIR ?= $(APPLICATION_DIR)/contrib/saline
```

this should be fluid properties module instead of APP_DIR ?

---

Hm. I can double check when I'm back. I thought that get changed multiple times during build to whatever it needs to be...

---

this should be changed to
```suggestion
SALINE_DIR ?= $(MOOSE_DIR)/modules/fluid_properties/contrib/saline
```